tests/python/frontend/onnx/test_forward.py

```diff
@@ -2027,6 +2027,7 @@ def verify_pad_v11(indata, pads, mode="constant", value=0.0):
     verify_pad_v11(np.random.randn(2, 2).astype(np.float32), [0, 1, 0, 0], "constant", 0.0)
     verify_pad_v11(np.random.randn(2, 3).astype(np.float32), [1, 0, 0, 1], "constant", 0.0)
     verify_pad_v11(np.random.randn(3, 2).astype(np.float32), [0, 0, 1, 0], "constant", 5.0)
+    verify_pad_v11(np.random.randn(3, 2).astype(np.float32), [0, 0, 1, 0], "constant", False)
```

Comment:
Is setting `False` here (as opposed to `None`) reproducing the same issue this patch is fixing?

Reply:
No, But I can't add a test case with `None`.
First, In the doc `test_forward.py   test_pad`, The `value` is a default number,can't set `None`.
`def verify_pad_v11(indata, pads, mode="constant", value=0.0)`
Then, the input node 'constant_value' is a `TensorProto.FLOAT`. not `None` too.
Can you give me some suggestions to add a test case with 'None'?  Thanks very much.